### PR TITLE
Orgs and Projects events source

### PR DESCRIPTION
**Pull request by julienmachon**
SO, you can now subscribe to Org and Project events:

```typescript
const eventSource = Project.subscribe({
  onProjectCreated: (projectCreated) => /* do whatever you want */,
  onProjectUpdated: (projectCreatedEvent) => /* ... */,
  onProjectDeprecated: (projectDeprecated) => /* ... */,
})
```

and the same for organisations:

```typescript
const eventSource = Organization.subscribe({
  onOrgCreated: (projectCreated) => /* do whatever you want */,
  onOrgUpdated: (projectCreatedEvent) => /* ... */,
  onOrgDeprecated: (projectDeprecated) => /* ... */,
})

// when your done
eventSource.close()
```

That means we can integrate this to nexus-web and get live notifications

### CODE REVIEW

File: nexus-sdk-js/src/Project/utils.ts [hunk 1]
```diff
@@ -127,3 +134,32 @@ export async function deprecateProject(
     throw new Error(error);
   }
 }
+
+export function subscribe(listeners: ProjectEventListeners): EventSource {
+  const event: EventSource = getEventSource('/projects/events');
+  //
+  // set event listeners
```
Comment (mdupont) [inline nexus-sdk-js/src/Project/utils.ts]: Wrong comment style

File: nexus-sdk-js/src/utils/events.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+// @ts-ignore
+import EventSource = require('eventsource');
+import store from '../store';
+
+// TODO: speak to backend to figure out token
+const getHeaders = (): EventSourceInit => {
```
Comment (mdupont) [inline nexus-sdk-js/src/utils/events.ts]: I thought we already had a generic method to add auth token to a request. If not, better make it something generic inside utils/http.ts

Comment (julienmachon) [reply nexus-sdk-js/src/utils/events.ts]: Headers for an` HTTP` request or `EventSource` are completely different unfortunately. There aren't really header for ES, you just give it a config: https://developer.mozilla.org/en-US/docs/Web/API/EventSource/EventSource

That's why I put a wee TODO there, making sure we find a solution with the backend team

File: nexus-sdk-js/src/Project/utils.ts (hunk 1 above)
Comment (julienmachon) [reply nexus-sdk-js/src/Project/utils.ts]: Fixed in 394299cab09907ae0f3c434bd34f10188b29a7a9

File: nexus-sdk-js/src/utils/events.ts (hunk 2 above)
Comment (mdupont) [reply nexus-sdk-js/src/utils/events.ts]: I see

**Pull request merged**